float32 dxl_0
float32 dxl_1
float32 dxl_2
float32 dxl_3
float32 dxl_4
float32 dxl_5
